src/app/wallets/get-csv-for-wallets.ts

```diff
@@ -1,12 +1,12 @@
 import { CSVAccountExport } from "@core/csv-account-export"
-import { accountPath } from "@services/ledger/accounts"
+import { walletPath } from "@services/ledger/accounts"
 
 export const getCSVForWallets = async (
   walletIds: WalletId[],
 ): Promise<string | ApplicationError> => {
   const csv = new CSVAccountExport()
   for (const walletId of walletIds) {
-    await csv.addAccount({ account: accountPath(walletId) })
+    await csv.addAccount({ account: walletPath(walletId) })
```

Comment:
This seems to still be on the wrong side of the core / ledger boundary - is that right?
Not saying it needs to be done in this PR just asking to check if we have a shared understanding.

Reply:
that is correct. `walletPath` should not be used outside ledger (eventually)

Reply:
Right so probably this use case is not on `app/wallets` but `app/ledger`